src/browserlib/extract-cssdfn.js

```diff
@@ -50,10 +50,15 @@ const dfnLabel2Property = label => label.trim()
 const extractTableDfn = table => {
   let res = {};
   const lines = [...table.querySelectorAll('tr')]
-    .map(line => Object.assign({
-      name: dfnLabel2Property(line.querySelector(':first-child').textContent),
-      value: line.querySelector('td:last-child').textContent.trim().replace(/\s+/g, ' ')
-    }));
+    .map(line => {
+      const cleanedLine = line.cloneNode(true);
+      const annotations = cleanedLine.querySelectorAll("aside");
+      annotations.forEach(n => n.remove());
+      return Object.assign({
```

Comment:
Call to `Object.assign` can be dropped, I think. It was only there to avoid adding a return statement with `=>`.